StoreAPI/Models/OrderProduct.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StoreAPI.Models
+{
+    public class OrderProduct

```

Comment:
OrderProduct entity should contain OrderId, and info about Product (Id, quantity). Address should not be here. Address must be in Order entity.